cfme/fixtures/cli.py

```diff
@@ -146,6 +178,33 @@ def app_creds_modscope():
     }
 
 
+def get_puddle_cfme_version(repo_file_path):
+    """Gets the version of cfme package in the the [cfme] repo on repo_file_path
+    """
+    namespaces = {'repo': 'http://linux.duke.edu/metadata/repo',
+                 'common': 'http://linux.duke.edu/metadata/common'}
```

Comment:
what is this URL?

Reply:
It is from the *-primary.xml file (which is the package metadata file) in the repodata dir in the repo:
```
<?xml version="1.0" encoding="UTF-8"?>
<metadata xmlns="http://linux.duke.edu/metadata/common" xmlns:rpm="http://linux.duke.edu/metadata/rpm" packages="154">
<
```